src/current/_includes/releases/v23.2/v23.2.0-beta.3.md

```diff
@@ -0,0 +1,43 @@
+## v23.2.0-beta.3
+
+Release Date: December 13, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-0-beta-3-general-changes">General changes</h3>
+
+- Updated Go version to 1.21.3. [#116098][#116098]
```

Comment:
The link for this is not rendering. I think because there is no github link at the end of the file for this PR